openadapt/record.py

```diff
@@ -197,23 +206,44 @@ def process_events(
                 num_video_events.value += 1
         elif event.type == "window":
             prev_window_event = event
+        elif event.type == "browser":
+            prev_browser_event = event
         elif event.type == "action":
             if prev_screen_event is None:
                 logger.warning("Discarding action that came before screen")
                 continue
             if prev_window_event is None:
                 logger.warning("Discarding input that came before window")
                 continue
+            if prev_browser_event is None:
+                logger.warning("Discarding input that came before browser")
+                continue
+
```

Comment:
At the very least this needs to be conditional on `config.REORD_BROWSER_EVENTS`, otherwise everything will be discarded whenever that value is False.

However even if that value is True, I'm not sure we want to discard. How about logging a warning instead?

Reply:
I commited this suggestion: https://github.com/OpenAdaptAI/OpenAdapt/pull/744#discussion_r1691440931

Reply:
You are right here to `not to skip if it is true`